pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java

```diff
@@ -1,23 +1,5 @@
 package org.jboss.pnc.core.builder;
```

Comment:
That's an issue because IDEs are automatically arranging imports. We need need to share the same formatter settings to avoid it.